package.json

```diff
@@ -52,7 +52,10 @@
     "watch": "tsc -watch -p ./",
     "pretest": "npm run prettier && npm run lint && tsc --project ./ts-test.json",
     "lint": "eslint src --ext ts",
-    "test": "node ./out_test/test/runTest.js"
+    "test": "node ./out_test/test/runTest.js",
+    "preinstall": "npx vsce package",
+    "install": "code --install-extension ./vscode-markdown-paste-image-*.vsix"
```

Comment:
This will cause `npm i` run code in testing, should be change to other name for ex. `dev:install`

![image](https://user-images.githubusercontent.com/10979091/162346985-547ad229-95f0-4756-8f04-3f311e719ede.png)